packages/fiori/src/Wizard.ts

```diff
@@ -771,15 +784,15 @@ class Wizard extends UI5Element {
 	}
 
 	get slottedSteps() {
-		return this.getSlottedNodes("steps");
+		return this.getSlottedNodes("steps") as Array<WizardStep>;
 	}
 
 	get contentDOM() {
```

Comment:
since we don't have conditionally rendering we can use ! here at the end of the line instead of adding it on every place where this getter is used